extensions/amp-subscriptions/0.1/amp-subscriptions.css

```diff
@@ -107,6 +107,34 @@ html:not(.i-amphtml-subs-grant-yes) .i-amphtml-subs-dialog-close-button {
   display: none;
 }
 
+.subscriptions-progress {
+  height: 2px;
+  background-color: #ccc;
+  position: relative;
+  margin: 8px;
+  overflow: hidden;
+}
+
+.subscriptions-progress:after {
+  content: '';
+  background-color: #2196F3;
+  height: 2px;
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 20%;
+  animation: progress 1500ms ease-in-out infinite;
+}
+
+@keyframes progress {
```

Comment:
The keyframes name should be fully namespaced and private. E.g. `i-amphtml-subs-kf-progress` or such.